frontend/src/browser/users/EditUser.js

```diff
@@ -626,27 +674,26 @@ export class EditUser extends Component {
                     />
 
                     {mode !== 'profile' ?
-                      <Field
-                        name="userState"
-                        component={this.renderSelect}
-                        label={`${formatMessage(messages.userState)}`}
-                      >
-                        <option value={'active'}>
-                          {formatMessage(messages.activeUserState)}
-                        </option>
-                        <option value={'inactive'}>
-                          {formatMessage(messages.inactiveUserState)}
-                        </option>
-                        <option value={'temporarySuspended'}>
-                          {formatMessage(messages.temporarySuspendedUserState)}
-                        </option>
-                        <option value={'temporarySuspended'}>
-                          {formatMessage(messages.expelledUserState)}
-                        </option>
-                        <option value={'temporarySuspended'}>
-                          {formatMessage(messages.endedUserState)}
-                        </option>
-                      </Field>
+                      <div>
+                        <Field
+                          name="state"
+                          component={this.renderSelect}
+                          label={`${formatMessage(messages.userState)}`}
+                        >
+                          {userStates ? Object.keys(userStates).map((key) =>
+                            <option value={userStates[key]}>
+                              {formatMessage(messages[`${userStates[key]}UserState`])}
+                            </option>
+                          ) : ''}
+                        </Field>
+
+                        <Field
+                          name="monthlySchoolFee"
```

Comment:
podla mna v eurach, ale v DB urcite storovat v centoch

2016-10-28 21:50 GMT+02:00 filek <notifications@github.com>:

> In frontend/src/browser/users/EditUser.js
> <https://github.com/Nexteria/Nextis/pull/103#discussion_r85598474>:
>
> > -                      </Field>
> > +                      <div>
> > +                        <Field
> > +                          name="state"
> > +                          component={this.renderSelect}
> > +                          label={`${formatMessage(messages.userState)}`}
> > +                        >
> > +                          {userStates ? Object.keys(userStates).map((key) =>
> > +                            <option value={userStates[key]}>
> > +                              {formatMessage(messages[`${userStates[key]}UserState`])}
> > +                            </option>
> > +                          ) : ''}
> > +                        </Field>
> > +
> > +                        <Field
> > +                          name="monthlySchoolFee"
>
> ešte je tu problém, toto sa má zadávať v centoch alebo v eurách?
> momentálne je to v centoch, ale neviem či je to vyhovujúce, čo ty na to
> @ddeath <https://github.com/ddeath>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nexteria/Nextis/pull/103/files/9f022dc22ea8614cb03e6c401063d4b718c761ad#r85598474>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACa3oyRNa1n8Rx_cnKgNKb3nGSgdr2xnks5q4lH9gaJpZM4KeFxZ>
> .
>



-- 
S pozdravom Dušan Plavák